expansion/src/main/java/no/unit/nva/expansion/model/ExpandedDatabaseEntry.java

```diff
@@ -26,4 +25,6 @@ default String toJsonString() {
             throw new RuntimeException(e);
         }
     }
+
+    SortableIdentifier fetchIdentifier();
```

Comment:
non serializable method for fetching the identifier for all types of expanded database entries. One of the issues is that now DoiRequests and Messages do not have an ID because there is no way to GET a DoiRequest or a Message